docs/renderer.md

```diff
@@ -157,6 +157,33 @@ end function
 See that all the function callback does is call `enqueueUpdate()`, which will cause Kopytko to run the `render` function
 again, this time getting the new value from `m.top.itemContent.labelText` and applying the change later on.
 
+### KopytkoRoot and initKopytkoRoot
```

Comment:
I have a feeling that docs are missing broader context here. This and the previous paragraph does not give information on why I might need to use `initKopytkoRoot`. I might need it when I want to integrate a parent non kopytko based component with a child kopytko. Could we add clear explanation on that?

Reply:
>Also, I'd rename this paragraph into sth like "Using Kopytko components in the non-Kopytko environment" and make as a second-level paragraph.
this or the one above?

Reply:
Created new section as @pawelhertman suggested and moved there related sub-paragraphs.

Also, I've added an issue https://github.com/getndazn/kopytko-framework/issues/33 to extract separate docs for the integration.